### PR TITLE
refactor: add paginated auction name endpoints

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]
looks good for me

File: lib/ae_mdw/auction_bids.ex [hunk 1]
```diff
@@ -12,27 +12,70 @@ defmodule AeMdw.AuctionBids do
...
+
+  @spec top_auction_bid(plain_name(), boolean()) :: {:ok, auction_bid()} | :not_found
+  def top_auction_bid(plain_name, expand?) do
     case Mnesia.prev_key(@table, bid_top_key(plain_name)) do
       {:ok, auction_bid} ->
-        if(elem(auction_bid, 0) == plain_name, do: {:ok, render(auction_bid)}, else: :not_found)
+        if(elem(auction_bid, 0) == plain_name,
+          do: {:ok, render(auction_bid, expand?)},
+          else: :not_found
+        )
```
Comment (jyeshe) [inline lib/ae_mdw/auction_bids.ex:34]: nitpick to if do else without inliner `, do:`

File: lib/ae_mdw/txs.ex [hunk 2]
```diff
@@ -37,27 +37,24 @@ defmodule AeMdw.Txs do
...
     {block_hash, type, signed_tx, tx_rec} = Db.get_tx_data(tx_hash)
 
-    tx_map =
-      tx_rec
-      |> Format.to_raw_map(type)
-      |> put_in([:type], type)
+    header = :aec_db.get_header(block_hash)
+    %{"tx" => tx} = :aetx_sign.serialize_for_client(header, signed_tx)
 
     raw = %{
```
Comment (jyeshe) [inline lib/ae_mdw/txs.ex:43]: Could we put this transaction code change in a separate PR to help keep tracking of the issues? (if it doesn't really relate).

**Review by jyeshe — COMMENTED** [review]
Nice refactor. Very small changes.

The only suspects are these two guys (in mdw_height: 452273)
```
  1) test auctions get auctions with parameters by=expiration, direction=forward and limit=3 (Integration.AeMdwWeb.NameControllerTest)
     test/integration/ae_mdw_web/controllers/name_controller_test.exs:203
     ** (KeyError) key :scope not found in: %{cursor: nil, direction: :forward, expand?: false, limit: 3, order_by: :expiration}
     code: conn.assigns.scope, 0},
     stacktrace:
       test/integration/ae_mdw_web/controllers/name_controller_test.exs:215: (test)



  2) test auctions get auctions with default limit (Integration.AeMdwWeb.NameControllerTest)
     test/integration/ae_mdw_web/controllers/name_controller_test.exs:174
     ** (KeyError) key :scope not found in: %{cursor: nil, direction: :backward, expand?: false, limit: 10, order_by: :expiration}
     code: {NameController, :auctions, %{}, conn.assigns.scope, 0},
     stacktrace:
       test/integration/ae_mdw_web/controllers/name_controller_test.exs:180: (test)
```

File: lib/ae_mdw/auction_bids.ex (hunk 1 above)
Comment (sborrazas) [reply lib/ae_mdw/auction_bids.ex]: Good call. Done

File: lib/ae_mdw/txs.ex (hunk 2 above)
Comment (sborrazas) [reply lib/ae_mdw/txs.ex:43]: it does relate, because for every auction a transaction (last_bid) is rendered. This is why this change was introduced.

Comment (jyeshe) [reply lib/ae_mdw/txs.ex:43]: Good, it's not used by the tx_controller.

File: lib/ae_mdw/txs.ex [hunk 3]
```diff
@@ -37,27 +37,24 @@ defmodule AeMdw.Txs do
...
       tx_index: tx_index,
-      tx: tx_map
+      tx: tx
     }
 
     type
     |> Format.custom_raw_data(raw, tx_rec, signed_tx, block_hash)
-    |> update_in_if_present([:tx, :account_id], &render_id/1)
-    |> update_in_if_present([:tx, :name_id], &render_id/1)
-    |> update_in_if_present([:tx, :recipient_id], &render_id/1)
```
Comment (jyeshe) [inline lib/ae_mdw/txs.ex:60]: `account_id` appeared and `recipient_id` not shown like master :+1:

**Review by jyeshe — APPROVED** [review]
Code looking good and test passes.
